exercises/chapter12/src/Example/LSystem.purs

```diff
@@ -7,7 +7,7 @@ import Data.Array (concatMap, foldM)
 import Effect (Effect)
 import Graphics.Canvas (strokePath, setStrokeStyle, lineTo, moveTo,
                         getContext2D, getCanvasElementById)
-import Math as Math
+import Data.Number as Math
```

Comment:
I know this probably keeps the diff minimal, but I think it's more idiomatic to have the namespace match the module name, like
```suggestion
import Data.Number as Number
```

Reply:
@JordanMartinez and @ptrfrncsmrph thanks for the reviews.
When is it idiomatic to use the namespace and when is it idiomatic to remove it entirely? I'd think it would be if only a few functions from the library are used. I was wondering if I should just drop the namespace and import `(cos, sin, tau)`?

Reply:
I don't think there is an idiomatic usage of a namespace or not. Often times, the namespace is useful when you want to clarify _which_ function you're using. For example, I could do `length`, but it's not obvious _which_ length that is. If I use `Array.length` or `String.length`, it's pretty obvious.

I think keeping the `Number` namespace here is desirable because there are other versions of `cos` (e.g. `Data.Decimal (cos)`). It's also probably a smaller diff if just the namespace is changed.